charts/prometheus-postgres-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "v0.15.0"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 6.1.0
+version: 6.1.1
```

Please, raise the minor version number whilst adding a new feature/field.
  ```suggestion
version: 6.2.0
```